app/views/layouts/_generic.njk

```diff
@@ -1,17 +1,32 @@
 {% extends "template.njk" %}
 
 {% block head %}
-  <!--[if !IE 8]><!-->
-    <link rel="stylesheet" href="/public/app.css">
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/prism/1.6.0/themes/prism.min.css" integrity="sha256-vtR0hSWRc3Tb26iuN2oZHt3KRUomwTufNIf5/4oeCyg=" crossorigin="anonymous" />
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/prism/1.6.0/prism.min.js" integrity="sha256-Zb9yKJ/cfs+jG/zIOFL0QEuXr2CDKF7FR5YBJY3No+c=" crossorigin="anonymous"></script>
-  <!--<![endif]-->
-  <!--[if IE 8]>
-    <link rel="stylesheet" href="/public/app-ie8.css">
-  <![endif]-->
-  <!--[if lt IE 9]>
-    <script src="/vendor/html5-shiv/html5shiv.js"></script>
-  <![endif]-->
+
+  {% if legacy %}
```

Comment:
You also need to pull in the stylesheets from GOV.UK Template separately:

- https://govuk-frontend-review-pr-1279.herokuapp.com/vendor/govuk_template/govuk-template.css
- https://govuk-frontend-review-pr-1279.herokuapp.com/vendor/govuk_template/govuk-template-print.css
- https://govuk-frontend-review-pr-1279.herokuapp.com/vendor/govuk_template/govuk-template-ie8.css)

Reply:
Ah this might explain why there fonts appear to be different when you toggle between the two modes

Reply:
If you mean the component page, I believe that's happening because the markup for the headings is weird (the heading class is on a parent div, not the heading, so the reset on the heading wins)